docs/content/1.get-started/3.providers.md

```diff
@@ -26,12 +26,17 @@ For example, when using `<NuxtImg src="/nuxt-icon.png" />`, it should be placed
 For more information, you can learn more about the [public directory](https://nuxt.com/docs/guide/directory-structure/public).
 
 ::callout
-Image stored in the `assets/` directory are **not** proccessed with Nuxt Image because those images are managed by webpack.
+Image stored in the `assets/` directory are **not** processed with Nuxt Image because those images are managed by Webpack.
```

Comment:
```suggestion
Image stored in the `assets/` directory are **not** processed with Nuxt Image because those images are managed by your bundler (such as Vite or webpack).
```